pom.xml

```diff
@@ -265,19 +253,6 @@
       <artifactId>commons-logging</artifactId>
       <scope>provided</scope>
     </dependency>
-    <dependency>
-      <groupId>org.kohsuke</groupId>
-      <artifactId>access-modifier-annotation</artifactId>
-      <scope>provided</scope>
-      <optional>true</optional>
-      <exclusions>
-        <exclusion>
-          <!-- provided by Jenkins core -->
-          <groupId>org.jenkins-ci</groupId>
-          <artifactId>annotation-indexer</artifactId>
-        </exclusion>
-      </exclusions>
-    </dependency>
```

Comment:
`jenkins-core` (which we depend on above) already depends on `access-modifier-annotation` transitively, so this is redundant. Furthermore, it causes Enforcer errors because of Maven's "nearest definition" algorithm. Better to remove it and have plugins get the version corresponding to their core baseline.